### DISCUSSION TITLE
VGSNAG3.CFF

### DISCUSSION THREAD

**Discussion by MasterCodeIT** [jglim/CaesarSuite#43] [General]
Possible to enabled development mode?

**Comment by Feezex** [comment]
maybe in cbf?

**Reply by N0cynym** [reply]
Try `IOC_DeviceMode_Control`.

**Comment by MasterCodeIT** [comment]
Tried the IOC, no luck. 

The development mode routine request starts with 31 01 09999 xxxxxxxxx
